src/css/vocabulary.css

```diff
@@ -1819,6 +1852,50 @@ body > footer .license svg {
     --underline-background-color: var(--vocabulary-brand-color-soft-turquoise);
 }
 
+/* scroll to top */
```

Comment:
There are several changes here related to the scroll-to-top feature that need to be removed.

PRs should be concisely related to the Issue they are addressing only. Please remove this.